level1c4pps/__init__.py

```diff
@@ -134,22 +139,24 @@ def get_band_encoding(dataset, bandnames, pps_tagnames, chunks=None):
                 'add_offset': 0.0}
         if chunks is not None:
             enc['chunksizes'] = chunks
-    if name in ['lon', 'lat']:
+    elif name in ['lon', 'lat']:
```

Comment:
I would like to have an if here and not elif. To make it possible to add more id_tags for example to lat/lon without changing this function. The qual_flags  (gac) I think also have the id_tag attribute, and with this update they would not get the correct encoding. However my test was to badly written to capture it.

I should have defined qual_f as:
qual_f = mock.MagicMock(attrs={'name': 'qual_flags',
                                                     'id_tag': 'qual_flags'})

Reply:
Oh, sorry! I have reverted the changes as you suggested.